docs/localization-ru.md

```diff
@@ -0,0 +1,33 @@
+# Russian localization notes
+
+## Style guide
+
+- Буква Ё/ё используется
+- Знак препинания `;` заменяется на точку
+- Не стоит в точности повторять структуру английского сообщения, если она неестественна для русского языка.
+  В частности избегайте конструкции `X является Y` (`класс Foo является абстрактным` -> `класс Foo абстрактный`)
+- По возможности делайте структуру сообщения проще
+
+## Глоссарий
```

Comment:
we should probably structure this file such that localization notes for other languages can be added in the future

```suggestion
# Localization notes

pyright's localizations seem to come from some sort of automated system maintained by the pylance team rather than by actual humans, which results in many of the translations being very bad. this also means they do not accept contributions to fix said translations. here are some guidelines for contributors who would like to help improve the translations in basedpyright.

## general guidelines

- in cases where a word is difficult to translate but it refers to the name of a symbol, leaving it in english and using backticks makes more sense than attempting to translate it. for example, if the english error message says "cannot assign to Final variable" then the translation could just be "невозможно присвоить переменной `Final`" instead of trying to use a russion word with a similar meaning to "final".
- basedpyright is maintained by developers who only speak english so we cannot reliably review your translations, so it would be helpful if you could get another developer who also speaks your language to review your changes as well.
- we do not use any automated systems to translate our new rules that are specific to basedpyright, so they currently do not have any translations at all (see [this issue](https://github.com/DetachHead/basedpyright/issues/81)). providing translations for those would be greatly appreciated.

## specific languages

### Russian

#### Style guide

- Буква Ё/ё используется
- Знак препинания `;` заменяется на точку
- Не стоит в точности повторять структуру английского сообщения, если она неестественна для русского языка.
  В частности избегайте конструкции `X является Y` (`класс Foo является абстрактным` -> `класс Foo абстрактный`)
- По возможности делайте структуру сообщения проще

#### Глоссарий
```